### DISCUSSION TITLE
+How to Input Unicode Characters in Input Remapper

### DISCUSSION THREAD

**Discussion by Lady-Fitzgerald** [sezanzeb/input-remapper#735] [Q&A]
I have an old Logitech keyboard I want to use with my Linux Mint 21.1 Cinnamon install so I can take advantage of the 18 G Keys (separate macro keys) to quickly type non-keyboard characters, such as à, and symbols, such as °.

I've tried using Input Remapper (https://github.com/sezanzeb/input-remapper) 1.4.0 (the version in the repository) but haven't any luck so far.

I can get keyboard characters to work with a G key but there's no point in that. However, I have not been able to get other characters and symbols, such as the ones I mentioned, to work with a G key using their Unicodes, such as CTRL+Shift+u+00b0 for °. I've tried to follow the directions on the programs GitHub page but I can't make any sense of them (it doesn't help that I'm a septuagenarian with ADHD).

Can someone help me with this. If I can get the correct syntax for entering one character or symbol, such as the ° symbol, using its Unicode. Because of my ADHD, I need simple, step by step directions.

Thanks!

**Comment by Lady-Fitzgerald** [comment]
I tried removing the older version and installing the latest one. Now I'm even more confused.

**Comment by Lady-Fitzgerald** [comment]
Someone on the Linux Mint forums far more knowledgeable than me finally figured out for me how to get it to work.

**Reply by montanajava** [reply]
Lady-Fitzgerald, could you perchance add a few lines as to how you did it, or a link to said forum?  I am trying to do the same thing.

**Comment by jonasBoss** [comment]
Unicode characters are probably not going to work. See [Limitations](https://github.com/sezanzeb/input-remapper/blob/main/readme/usage.md#limitations)

**Comment by Lady-Fitzgerald** [comment]
Uh, I replied that they do work, and work well, then closed the thread.

__________________________________________________________________________________________________________________

Sent from my PC using a real keyboard!
 

    On Tuesday, July 11, 2023 at 05:21:41 AM MST, jonasBoss ***@***.***> wrote:  
 
 


Unicode characters are probably not going to work. See Limitations

**Comment by Lady-Fitzgerald** [comment]
I don't know what else I could add. I just followed billyswong's directions.

__________________________________________________________________________________________________________________

Sent from my PC using a real keyboard!
 

    On Saturday, August 26, 2023 at 03:36:33 AM MST, montanajava ***@***.***> wrote:  
 
 


Lady-Fitzgerald, could you perchance add a few lines as to how you did it, or a link to said forum? I am trying to do the same thing.

**Reply by montanajava** [reply]
Okay, Lady-Fitzgerald, thanks. For others, the reference to billyswong can be found here:

https://forums.linuxmint.com/viewtopic.php?t=399740

The solution seems to be that Linux supports inserting Unicode via  Control_L- Shift_L-U and then typing the Unicode code and then Enter.

So far so good. It just doesn't work for me. I will create a new issue requesting instruction clarification as I regard this as a super helpful feature.

**Comment by fazcue** [comment]
This is how I was able to make it work:

`modify(Control_L, modify(Shift_L, key(u))).key(2).key(2).key(2).key(8).key(Return)`

It will print the unicode symbol "∨", which is U+2228

Another example with the OP Unicode:

`modify(Control_L, modify(Shift_L, key(u))).key(0).key(0).key(b).key(0).key(Return)`

It will print the unicode symbol "°", which is U+00b0
